website/content/api-docs/system/remount.mdx

```diff
@@ -6,16 +6,32 @@ description: >-
   endpoint.
 ---
 
-# `/sys/remount`
+# Remount ( API )
 
-The `/sys/remount` endpoint is used remount a mounted backend to a new endpoint.
+The Remount documentation details the endpoints required to trigger and monitor the status of a remount operation.
 
 ## Move Backend
 
-This endpoint moves an already-mounted backend to a new mount point. 
+The `/sys/remount` endpoint moves an already-mounted backend to a new mount point. This works for both secret engines
+and auth methods.
+
+The endpoint can be used to move the backend across namespaces as well, with the namespaces supplied as prefixes
+on the path arguments. For this case, the namespaces of both the source and destination mounts need to be 
+either the namespace in which the `/remount` endpoint is called, or children of it. Say, we have 3 namespaces 
+ns1/, ns1/ns2/ and ns1/ns3. We want to move the mount `secret` from ns1/ns2/secret to ns1/ns3/secret. 
+We can either invoke from the root namespace with the `from` argument as ns1/ns2/secret and `to` argument 
+as ns1/ns3/new-secret, or we can invoke it from the namespace ns1/, with `from` as ns2/secret and `to` 
+as ns3/new-secret.
+
+The remount operation can be a long running operation, hence the invocation of /remount triggers the start of 
```

Comment:
```suggestion
The remount operation may be a long-running operation, hence the invocation of /remount triggers at the beginning of 
```

Reply:
some of this [ec185fc](https://github.com/hashicorp/vault/pull/14314/commits/ec185fca4801fb73794fecd7dda77eea8a8e6d22)